### DISCUSSION TITLE
PatchTool 7.0 7/17

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#461] [Announcements]
相容性更新。

以下兩種方法擇一使用即可；請從release下方連結或內文連結下載檔案。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20240717.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20240717.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20240717.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- FFXIVChnTextPatch-20240628.jar：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20240717.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.3.0.4'>PatchTool 7.0 7/17</a>.</em>

**Comment by dnangel8042110** [comment]


**Reply by GpointChen** [reply]
是一進副本就閃退嗎？

**Reply by dnangel8042110** [reply]
結果應該是遊戲有壞檔 修復重載90G後就可以了 感恩

**Reply by KIDDTW** [reply]
![FINAL FANTASY XIV 2024_7_28 下午 03_27_49](https://github.com/user-attachments/assets/0477c8a6-1c9e-4155-8c24-568bce8c1901)
很多技能像這樣只翻了名字沒有技能說明翻譯 是正常的嗎

**Reply by GpointChen** [reply]
正常，部分技能有改動且涉及指令碼，為了避免導致遊戲崩潰，先使用原文。

**Comment by manlymeat7878** [comment]
![未命名](https://github.com/user-attachments/assets/99af3b03-fc53-4751-9e2b-f5a97df12dd7)
我是使用方法一，但會出現缺字替換成"="的狀況
想請問該怎麼解決這種問題？

**Reply by GpointChen** [reply]
預設字體包會缺部分大廳字型，在大廳出現=屬於正常現象。在正式遊戲中不會有問題。

如果介意，可以用release頁面中的高畫質字體包。

**Reply by manlymeat7878** [reply]
了解，非常感謝

**Comment by Bos-hunting** [comment]
前天更新了7.05，請問會有兼容版嗎？感謝作者。
